packages/components/src/components/accordion/accordion.ts

```diff
@@ -158,7 +193,7 @@ export default class SdAccordion extends SolidElement {
           class=${cx(
             'flex text-base gap-4 font-bold items-center cursor-pointer select-none px-4 py-3 focus-visible:focus-outline text-primary hover:bg-neutral-200 relative group'
           )}
-          role="button"
+          aria-label="header"
```

Comment:
Is this on purpose? What's the resoning,

Reply:
I changed this here because the accessibility tests were failing for sd-quickfact on these lines.